src/Razor/src/Microsoft.VisualStudio.RazorExtension/EmbeddedGrammars/aspnetcorerazor.tmLanguage.tmTheme

```diff
@@ -146,7 +146,7 @@
           <string>Identifier</string>
         </dict>
       </dict>
-
+<!--
```

Comment:
Maybe consider duplicating the comment on line 160 to all the commented out portions (i.e. `<!-- Temporarily removed because ...`)